fpdf/fpdf.py

```diff
@@ -4852,6 +4875,7 @@ def _default_file_id(self, buffer):
         # > When a file is first written, both identifiers shall be set to the same value.
         id_hash = hashlib.new("md5", usedforsecurity=False)  # nosec B324
         id_hash.update(buffer)
+        print(str(buffer))
```

Comment:
I think this `print()` call is hurting the CI pipeline log due to its huge size 😅